content/webapp/pages/whats-on/index.tsx

```diff
@@ -526,34 +549,38 @@ const WhatsOnPage: FunctionComponent<Props> = props => {
               </Space>
             </>
           )}
-          {period !== 'current-and-coming-up' && (
-            <SpacingSection>
-              <Space
-                $v={{ size: 'm', properties: ['padding-top', 'margin-bottom'] }}
-              >
-                <Layout gridSizes={gridSize12()}>
-                  <div
-                    style={{
-                      display: 'flex',
-                      alignItems: 'center',
-                      justifyContent: 'space-between',
-                    }}
-                  >
-                    <h2 className={font('wb', 2)}>Exhibitions and Events</h2>
-                    <span className={font('intb', 4)}>Free admission</span>
-                  </div>
-                </Layout>
-              </Space>
-              <ExhibitionsAndEvents
-                exhibitions={exhibitions}
-                events={eventsToShow}
-                links={[
-                  { text: 'View all exhibitions', url: '/exhibitions' },
-                  { text: 'View all events', url: '/events' },
-                ]}
-              />
-            </SpacingSection>
-          )}
+          {period !== 'current-and-coming-up' &&
```

Comment:
Same as above about the ANDing ... if we feel this is a common enough pattern in our codebase that someone should understand this, ignore me.